h/accounts/test/views_test.py

```diff
@@ -32,6 +32,8 @@ def configure(config):
     config.registry.registerUtility(SubmitForm, IProfileForm)
     config.registry.registerUtility(MagicMock(), IRegisterSchema)
     config.registry.registerUtility(MagicMock(), IRegisterForm)
+    config.registry.feature = MagicMock()
```

Comment:
Ok, <https://github.com/hypothesis/h/pull/2239>, I'll try to figure out what's going on here